hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/states/NodeStateMap.java

```diff
@@ -88,7 +88,7 @@ public void addNode(DatanodeDetails datanodeDetails, NodeState nodeState)
         throw new NodeAlreadyExistsException("Node UUID: " + id);
       }
       nodeMap.put(id, new DatanodeInfo(datanodeDetails));
-      nodeToContainer.put(id, new HashSet<>());
+      nodeToContainer.put(id, new ConcurrentHashMap().newKeySet());
```

Comment:
Nit: [newKeySet](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.html#newKeySet--) is `static`, so creating new `ConcurrentHashMap` instance is unnecessary.

```suggestion
      nodeToContainer.put(id, ConcurrentHashMap.newKeySet());
```

Reply:
Good point. I also make change to stateMap which has similar issue.